test/tests.js

```diff
@@ -521,6 +521,28 @@ describe("compiler", function () {
     isCallExprStmt(ast.body[5], "module", "export");
   });
 
+  it("should transform default export declaration to expression", function () {
+    import { compile } from "../lib/compiler.js";
+
+    var code = [
+      "export default class {}"
```

Comment:
Maybe give this class a name so that it's more ambiguous whether it's a declaration or an expression?